x-pack/plugins/infra/server/graphql/types.ts

```diff
@@ -212,7 +230,7 @@ export interface InfraNode {
 export interface InfraNodePath {
   value: string;
 
-  label: string;
+  label?: string | null;
```

Comment:
I think this is a left-over from a previously manual edit of the types. If this field was non-option in the schema https://github.com/elastic/kibana/blob/476a88e309320d0b35aa50cb86ab503e956cf9a0/x-pack/plugins/infra/server/graphql/nodes/schema.gql.ts#L17 it might save us some checks and special cases.